src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss

```diff
@@ -5,4 +5,8 @@
 @import 'node_modules/buying-catalogue-components/app/styles/main';
 
 // Buying catalogue login page styles
-@import 'login';
\ No newline at end of file
+@import 'login';
+
+.bc-t-bg--grey-4 {
+    background-color: #d8dde0
```

Comment:
please use nhs colours - https://service-manual.nhs.uk/design-system/styles/colour

Reply:
$color_nhsuk-grey-4

Reply:
Is it better using 
background-color: $nhsuk-border-color 

or do you want
background-color: $color_nhsuk-grey-4

Reply:
I prefer this as it is explicit:  $color_nhsuk-grey-4